src/helpers/api.ts

```diff
@@ -0,0 +1,17 @@
+import { Product } from '../types/Product';
+
+// eslint-disable-next-line max-len
+const API_URL = 'https://mate-academy.github.io/react_phone-catalog/api/products.json';
+
+// function wait(delay: number) {
```

Comment:
Don't forget to delete comments. Also try not to disable eslint but fix its problem